### ISSUE TITLE
Playwright Bug

### ISSUE THREAD

**Issue by krishnakolluneocol** [SFDO-Tooling/CumulusCI#3498]
### Describe the bug

Hello,

When I run the example test mentioned here https://cumulusci.readthedocs.io/en/latest/robot-playwright.html, I get this error:

_Suite setup failed: Timed out waiting for a lightning page_

![image](https://user-images.githubusercontent.com/93388033/213237387-6ac907ff-251e-4136-be4b-9972a47ff97c.png)

<img width="643" alt="image" src="https://user-images.githubusercontent.com/93388033/213237624-831ffe51-5ecc-445e-a762-c9606702fd32.png">

This is the page that opened:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/93388033/213238815-baa44510-7cff-4b64-b657-cf8349120a83.png">

### Reproduction steps

1. Authorize cumulusci to a sandbox
2. Run the script mentioned here: https://cumulusci.readthedocs.io/en/latest/robot-playwright.html
3. Review errors provided in script
n. ...

### Your CumulusCI and Python versions

CumulusCI version: 3.71.0 (/Users/kollu/.local/bin/cci)
Python version: 3.11.1 (/Users/kollu/.local/pipx/venvs/cumulusci/bin/python)
You have the latest version of CumulusCI ⛅

### Operating System

MacOs Ventura Version 13.0

### CumulusCI installation method

pipx

**Comment by GeekStewie** [comment]
We hit this as well but you can turn off the wait. See the example below:

```
*** Settings ***
Resource          cumulusci/robotframework/SalesforcePlaywright.robot
Suite Setup       Run keywords
...               Open test browser    wait=false
...               AND Set browser timeout    60 seconds
Suite Teardown    Close browser
```

**Comment by jstvz** [comment]
Fixed in #3503
